completed major tasks

StephenSGambill opened the pull request.

StephenSGambill committed:
completed major tasks

StephenSGambill committed:
completed major tasks

StephenSGambill committed:
Merge branch 'major_tasks' of github.com:StephenSGambill/jr_rails into major_tasks

StephenSGambill committed:
fixed date off 1 day problem

StephenSGambill committed:
made input fields for contacts required

brianburridge commented on app/views/application/home.html.erb:
```diff
@@ -18,53 +18,61 @@
       <div class="mx-auto max-w-7xl sm:px-6 lg:px-8">
         <div class="mx-auto flex max-w-2xl flex-col gap-16 bg-white/5 px-6 py-16 ring-1 ring-white/10 sm:rounded-3xl sm:p-8 lg:mx-0 lg:max-w-none lg:flex-row lg:items-center lg:py-20 xl:gap-x-20 xl:px-20">
-          <img class="h-96 w-full flex-none rounded-2xl object-cover shadow-xl lg:aspect-square lg:h-auto lg:max-w-sm" src="https://images.unsplash.com/photo-1519338381761-c7523edc1f46?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=800&q=80" alt="">
+          <img class="h-96 w-full flex-none rounded-2xl object-cover shadow-xl lg:aspect-square lg:h-auto lg:max-w-sm" src="https://media.licdn.com/dms/image/D4E03AQGvE_e3D8VbwA/profile-displayphoto-shrink_800_800/0/1693326236059?e=2147483647&v=beta&t=o5yKBfQVuGu4um-Dw-tGuGpLstTUKp4SXTEp8nH-0NA" alt="">
```
For me, this image is not showing.

StephenSGambill committed:
updated profile pic link to cloudinary

StephenSGambill commented:
Hey there Brian,

That’s strange, as the image link was working on mine. 

Nevertheless, I changed it to a Cloudinary based image, so hopefully that works for you. Let me know if it doesn’t.

Also, in my code, I removed the unSplash image, but it appears in the code you included below. Is that because of a merge issue?

Let me know if there is anything else.

Stephen

> On Sep 25, 2023, at 11:04 AM, Brian Burridge ***@***.***> wrote:
> 
> 
> @brianburridge requested changes on this pull request.
> 
> In app/views/application/home.html.erb <https://github.com/FamilyFirstAPD/jr_rails/pull/1#discussion_r1336092607>:
> 
> > @@ -18,53 +18,61 @@
>      <div class="relative isolate">
>        <div class="mx-auto max-w-7xl sm:px-6 lg:px-8">
>          <div class="mx-auto flex max-w-2xl flex-col gap-16 bg-white/5 px-6 py-16 ring-1 ring-white/10 sm:rounded-3xl sm:p-8 lg:mx-0 lg:max-w-none lg:flex-row lg:items-center lg:py-20 xl:gap-x-20 xl:px-20">
> -          <img class="h-96 w-full flex-none rounded-2xl object-cover shadow-xl lg:aspect-square lg:h-auto lg:max-w-sm" src="https://images.unsplash.com/photo-1519338381761-c7523edc1f46?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=800&q=80" alt="">
> +          <img class="h-96 w-full flex-none rounded-2xl object-cover shadow-xl lg:aspect-square lg:h-auto lg:max-w-sm" src="https://media.licdn.com/dms/image/D4E03AQGvE_e3D8VbwA/profile-displayphoto-shrink_800_800/0/1693326236059?e=2147483647&v=beta&t=o5yKBfQVuGu4um-Dw-tGuGpLstTUKp4SXTEp8nH-0NA" alt="">
> For me, this image is not showing.
> 
> —
> Reply to this email directly, view it on GitHub <https://github.com/FamilyFirstAPD/jr_rails/pull/1#pullrequestreview-1642514639>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A3UIMTAJOOOYM5RM37LNF4LX4GTRRANCNFSM6AAAAAA5ERAXYU>.
> You are receiving this because you authored the thread.
>

brianburridge commented:
> Hey there Brian, That’s strange, as the image link was working on mine. Nevertheless, I changed it to a Cloudinary based image, so hopefully that works for you. Let me know if it doesn’t. Also, in my code, I removed the unSplash image, but it appears in the code you included below. Is that because of a merge issue? Let me know if there is anything else. Stephen

The image looks good now. Maybe something in my browser was blocking it from the previous provider.

I think that's just the way Github was showing the PR comparison. It's not still in the code.
